tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py

```diff
@@ -155,3 +157,18 @@ def get_modified_connectors(modified_files: Set[str]) -> Set[Connector]:
         if file_path.startswith(SOURCE_CONNECTOR_PATH_PREFIX) or file_path.startswith(DESTINATION_CONNECTOR_PATH_PREFIX):
             modified_connectors.append(Connector(get_connector_name_from_path(file_path)))
     return set(modified_connectors)
+
+
+def pipeline_command(func):
```

Comment:
I think you could achieve the same thing by creating a class inheriting from `click.Command` and overriding its `invoke`command:
```
@test.command(cls=PipelineCommand, help="Run unit tests for the metadata service library.")
```
It's something I did for `octavia-cli` [here](https://github.com/airbytehq/airbyte/blob/master/octavia-cli/octavia_cli/base_commands.py#L10).

Reply:
Great way to do this, will update